docs/requirements.rst

```diff
@@ -72,3 +74,19 @@ Additional Suggested Software
 
 - You may want to use autossh, available in various package management
   systems
+- If you are using systemd, sshuttle can notify it when the connection to
+  the remote end is established and the firewall rules are installed
```

Comment:
Should end the sentence with a full stop or colon. Yes, I know, looks line line 76 is wrong too.